src/components/Person/Person.jsx

```diff
@@ -1 +1,15 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name: personName, age, sex, isMarried, partnerName } = person;
+  const partnerStatus = `${partnerName} is my ${sex === 'm' ? 'wife' : 'husband'}`;
+  const relationStatus = isMarried ? partnerStatus : 'I am not married';
+
+  return (
+    <>
```

Comment:
Add tag section here and remove it in App component
```suggestion
    <section className="Person">
```